discourse.Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM ubuntu:focal
 
 # Used in Launchpad OCI Recipe build to tag the image.
-LABEL org.label-schema.version=v2.7.10
+LABEL org.label-schema.version=v2.7.13
```

Comment:
latest version in 2.7 from github. https://github.com/discourse/discourse/tags?after=v2.8.4

Reply:
Perhaps it's worth testing if https://github.com/canonical/discourse-k8s-operator/pull/43 does still crash and upgrade to the last version 2 that fixes all the existing critital and high vulnerabilities detected by trivy

Reply:
Sure, let me move the tests for creating category to #43. Would this be close to merging? since this is high priority fix, I was hoping to get it merged within today :)

Reply:
I've just pushed some changes. I suggest we see if the integration tests pass before doing anything.

Reply:
Sure, i'll create another branch and create a PR towards #43 :)